fhevm/tfhe.go

```diff
@@ -18,9 +18,13 @@ package fhevm
 
 /*
 #cgo linux CFLAGS: -O3 -I../tfhe-rs/target/release
-#cgo linux LDFLAGS: -L../tfhe-rs/target/release -l:libtfhe.a -lm
```

Comment:
what's the reason of switching to dynamic linking?

Reply:
On this part, I follow @IceTDrinker tutorial.
I replaced `-l:libtfhe.a` with `ltfhe` (like on Darwin) because it just didn't work on Linux otherwise.

Reply:
🤔

Reply:
try to keep -l:libtfhe.a

and change -ltfhe_c_api_dynamic_buffer to -l:libtfhe_c_api_dynamic_buffer.a

Reply:
try a single line of flags perhaps ?
```#cgo linux LDFLAGS: -L../tfhe-rs/target/release -l:libtfhe.a -L../tfhe-rs/target/release/deps -l:libtfhe_c_api_dynamic_buffer.a -lm```

Reply:
With two lines, it was what we did together and it didn't work for dynamic buffer (`-l:libtfhe_c_api_dynamic_buffer.a`). I try with one line :)

Reply:
Yep I know :) trying to debug with you :D